Automattic/wp-calypso
# Logged-in Site Performance: Refresh the report if the refresh interval has been passed

Related to #

## Proposed Changes

* Adds a banner that calls out stale report and give user the ability to retest

Desktop
![image](https://github.com/user-attachments/assets/45a5577c-caef-45f5-af28-6623f95aedd1)

Mobile
![image](https://github.com/user-attachments/assets/9aae7c5a-5bbb-4d22-a1b4-14b723155842)


## Why are these changes being made?


* So that user receives up to date report on their site's performance

## Testing Instructions



* Ensure you have a site that's performance report is more than 24 hours ago. If not hard-code the report timestamp just for testing.
* Go to the sites performance tab.
* Verify the banner is shown and your able to retest

## Pre-merge Checklist
